refocus-bdk-server.js

```diff
@@ -485,6 +485,22 @@ module.exports = (config) => {
     }, HEARTBEAT_TIMER);
   } // heartBeat
 
+  /**
+   * Checks RoomType bot list for bot triggering event
+   * @param {Object} event - refocusEvent
+   * @param {String} bot - botName
+   * @returns {boolean}
+   */
+  function isBotInstalledInRoom(event, bot) {
+    const botsInRoom = event.Room.RoomType.Bots;
+    const botsInstalled = [];
+    botsInRoom.forEach((bots) => botsInstalled.push(bots.name));
+    log.info(`bots installed in room: ${botsInstalled}`);
```

Comment:
I'd suggest avoiding `log.info` at bdk level unless it's absolutely necessary. 
For debugging purpose, we can perhaps use `log.debug`. 
Let me know what others think.

Reply:
yep, debbug will destroy vm space :)

Reply:
Thank you  @LongweiDeng, that was one of the things that I wanted to pass by you guys, the verbosity of Logs. I don't have strong feeling that this log needed, but I did use it for debugging purposes, is it okay, if I leave it as log.debug? Let me know otherwise.